bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -237,6 +239,7 @@ protected void initModel() {
             if (!isDecorative) {
                 if (StringUtils.isNotEmpty(linkURL)) {
                     linkURL = Utils.getURL(request, pageManager, linkURL);
+                    linkTarget = properties.get(PN_LINK_TARGET, false);
```

Comment:
@rubnig - does this property need to be dependent on the presence of a `linkURL`? Shouldn't it simply return whatever has been configured by an author? The rendering can make the decision about how to deal with the result.

Reply:
@richardhand I thought it is cleaner to only export the value when it has been explicitly set and an URL is configured. This property is tied to the link and only has meaning when a link is configured. That's why I am also using the @JsonInclude(value = JsonInclude.Include.NON_DEFAULT) annotation in the Image interface. It only adds the field in the JSON Exporter for SPA applications, when needed right now.

Reply:
I'm honestly not sure on this point - I understand the reasoning, but it's not a pattern we've used so far in the core components (as far as I'm aware) - currently, the implementations and their JSON exports are "honest" about what has been configured and don't link dependant properties in this way. 

Yes a link target doesn't make sense without a link, but perhaps the consumer should have access to all configuration and do with that what they will. On the other hand, as you say, it's a cleaner JSON result not to include unnecessary properties.

Reply:
@richardhand there already seems to be some kind of mechanism in place for the link. It already hides the link property from the JSON result, when a image is set to decorative. That's why I aligned with this functionality for the linkTarget. See the linked image in: [http://localhost:4502/content/core-components-examples/library/image/jcr:content/root/responsivegrid/demo_1473529393/component/image.model.json?wcmmode=disabled](http://localhost:4502/content/core-components-examples/library/image/jcr:content/root/responsivegrid/demo_1473529393/component/image.model.json?wcmmode=disabled)